PRIVACY.md

```diff
@@ -0,0 +1,21 @@
+# Semgrep Privacy Policy
+
+Semgrep collects usage data to help us improve the product. We send two types of data to r2c servers: scan data and findings data.
+
+## Scan data
+
+Scan data provide information on the environment and performance of Semgrep. They power dashboards, identify anomalies with the product, and are needed for billing. The classes of data included are:
+- Project identity (e.g. name, URL)
+- Scan environment (e.g. CI provider, OS)
+- Author identity (e.g. committer email)
```

Comment:
To avoid PII, is it possible to set this to be a hash of committer email?

Reply:
yes hashing with a salt is what we did in Bento and that worked well

Reply:
We currently use the author identity to render the Slack Webhook. If we want to explicitly ban the collection of author identities, we'll need to reconsider our integrations.

Note that collection != storage, so we can delineate between what we transmit and what we store.

Reply:
+1 on Note that collection != storage, so we can delineate between what we transmit and what we store.